corrected regular expressions and urllib call to run with Python 3.8.5

HSteeb opened the pull request:
- avoid `re.error: bad escape \c at position 56` etc.
- changed the strings of some `\\c` to raw strings
- took out some Unicode characters `\uFDE2` etc. out of raw strings
- corrected single `\z` in raw string to `\\z`
- Python 2 `urllib.urlopen` is now `urllib.request.urlopen`
=> Caution: this won't work with Python 2 (maybe someone knows enough Python to make this conditional)

DavidHaslam commented:
Hello Helmut.

If you wish to become more active with CrossWire, please subscribe to the sword-devel mailing list.

Details may be found in the relevant page on the [SWORD Developers’ Wiki](https://wiki.crosswire.org/)

Best regards 

David Haslam
CrossWire volunteer
Cheshire, UK

HSteeb commented:
Hello David,

thanks for the warm welcome! Maybe becoming more active with CrossWire
makes sense, I do not know much about it, yet.

I'm the initiator and organizing person of bible2.net (since ~2007).
Before that, I helped to spread the Moravian Watchwords in electronic
form, since the nineties :-) Your "GLosung" still supports the
watchwords and our new service, AFAIK.

Typically I get some Bible edition licensed from the publisher and need
to convert it into a verse-wise representation, as a basis for our daily
verses. This is when I'm desperately looking for scripts like usfm2osis.
I've already converted several Bibles from Word, even PDF, to a
structured XML. The XML is a simplified version of the verse-wise
Zephanja format. I added a mechanism to map the individual verse numbers
between the different Bible editions, using lists in XML, related with a
"reference" edition (the German Schlachter 2000).

So I likely have structured XML representations of Bible editions no
other person has. But for most of them, due to the licensing
restrictions, this won't be of any use to you. Would it be interesting
for you to get hold of freely distributable Bible editions in XML? I'd
have to check. We're especially also serving some minority languages /
dialects.

I observe that OSIS is quite frequently used at CrossWire, seems to be a
base for the work? On the web, I've also seen USX, USFX, whatever, but
hardly any tools to handle. This is really strange. Everybody seems to
be tied to USFM via Paratext. I guess I'll find more info about this in
your wiki, so I have to look.

Right now I am about preparing our verses for the BibleThainghlim from
Bibles International. I'm going to use one of the 4 (!) flavours of USFM
to OSIS conversion I have at hand now. The next step is to somehow
produce nice HTML from the OSIS, and to extract and publish the daily
verses, verse-wise.

So again thanks for caring, I'll check,
God bless

Helmut

Am 12.02.21 um 15:33 schrieb David Frank Haslam:
>
> Hello Helmut.
>
> If you wish to become more active with CrossWire, please subscribe to
> the sword-devel mailing list.
>
> Details may be found in the relevant page on the SWORD Developers’
> Wiki <https://wiki.crosswire.org/>
>
> Best regards
>
> David Haslam
> CrossWire volunteer
> Cheshire, UK
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/refdoc/Module-tools/pull/16#issuecomment-778230346>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIBZFHOW4DR4BHP4JP3XHI3S6U3Z5ANCNFSM4XNLZEMA>.
>

-- 
Helmut Steeb
hs2013@bible2.net

DavidHaslam commented:
Hi Helmut,

If you have a SWORD front-end such as Xiphos installed, you can use the diatheke utility to output a single verse or verse range quite easily.

NB. CrossWire doesn't use any texts from republishers such as Zefania that are not rigorous about copyright and distribution licenses.

We are always very careful about providing Bible modules made from an authentic text source and giving an accurate attribution.

Best regards 

David

DavidHaslam commented:
Hi Helmut,

The German dialect translations are new to me, and probably also to my German colleagues in CrossWire.

We'd certainly be interested in having those made into SWORD modules. Any help you can give in obtaining permissions from the publisher and/or translators would be much appreciated.

Best regards,

David
